indexer/database/bridge.go

```diff
@@ -0,0 +1,144 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jackc/pgtype"
+	"gorm.io/gorm"
```

Comment:
This looks useful https://gorm.io/docs/prometheus.html

Reply:
yea this will be good to export on our end